source/extensions/tracers/opentelemetry/tracer.cc

```diff
@@ -0,0 +1,143 @@
+#include "source/extensions/tracers/opentelemetry/tracer.h"
+
+#include <cstdint>
+#include <string>
+
+#include "envoy/config/trace/v3/opentelemetry.pb.h"
+
+#include "source/common/common/empty_string.h"
+#include "source/common/common/hex.h"
+
+#include "opentelemetry/proto/collector/trace/v1/trace_service.pb.h"
+#include "opentelemetry/proto/trace/v1/trace.pb.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace OpenTelemetry {
+
+// TODO: handle tracestate as well.
+static const absl::string_view kTraceParent = "traceparent";
+static const std::string kDefaultVersion = "00";
```

Comment:
This is a source of test flakiness because of the static initialization fiasco. Need to be `constexpr absl::string_view`.

Reply:
Nice, done.